ui/app/services/path-help.js

```diff
@@ -178,32 +177,37 @@ export default Service.extend({
   // Makes a call to grab the OpenAPI document.
   // Returns relevant information from OpenAPI
   // as determined by the expandOpenApiProps util
-  getProps(helpUrl, backend) {
-    // add name of thing you want
+  getProps(helpUrl, backend, pathName = '') {
     debug(`Fetching schema properties for ${backend} from ${helpUrl}`);
 
     return this.ajax(helpUrl, backend).then((help) => {
-      // paths is an array but it will have a single entry
-      // for the scope we're in
-      const path = Object.keys(help.openapi.paths)[0]; // do this or look at name
+      // help.openapi.paths is an array. Get the first unless the pathName is passed from caller
+      const path = pathName || Object.keys(help.openapi.paths)[0];
       const pathInfo = help.openapi.paths[path];
       const params = pathInfo.parameters;
       const paramProp = {};
 
       // include url params
       if (params) {
-        const { name, schema, description } = params[0];
```

Comment:
This is the crux of the change that fixes the UI issues. Before, we only cared about the first param for the path. Now that the backend mount path is also a path param, we instead iterate over all the path params, throw out `_mount_path` (because we already capture it as `backend`) and add the rest to the model